bundle/pom.xml

```diff
@@ -503,6 +503,11 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
The engine should rather be declared only as plugin dependency like outlined in https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html#select-engine-and-use-multiple-apis.

Reply:
I have made this change, but since the dependencyManagement section doesn't seem to apply to plugin dependencies, I had to repeat the desired JUnit version twice after all. It feels this is in the opposite direction than your other request.

We could always extract the desired JUnit version to a property, so it can be reused in different parts of the POM even without relying on the dependency management feature.

Reply:
Indeed, but I prefer not bloating the test classpath with the engines for no reason. So instead of referencing `junit-jupiter` I would recommend to only depend on its relevant individual transitive dependencies, i.e. `junit-juper-api` and `junit-jupiter-params` (https://github.com/junit-team/junit5/blob/main/junit-jupiter/junit-jupiter.gradle.kts) with scope `test`. Although the jupiter engine is just having scope `runtime` in `junit-jupiter`, this will be transitively converted to scope `test` (https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope).

Then both engines are only extending the surefire classpath. Introducing a common property name for easily updating all junit5 dependencies would be good though.

Reply:
I have pushed the requested changes. Does it look good now?

Reply:
Looks great, thanks a lot.